### PR TITLE
Fix fzf_binary_preview command for fish shell 

### PR DESCRIPTION
**Pull request by eddyekofo94**
Some auto formatting on the README but it shouldn't be too bad.
 fixes the #124

### CODE REVIEW

**Comment by yuki-yano** [comment]
The docs and settings are too complicated, so I think the current situation is fine.
Are the tips in the README insufficient?

```
set shell=/bin/zsh
let $SHELL = "/bin/zsh"
```

**Comment by judewang** [comment]
There are something I have no idea why. If I set let up fzf-preview by adding this line:

```vim
g:fzf_preview_if_binary_command = "test (file -b --mime-encoding {}) = 'binary'"
```

I got a very short loading time when trying moving cursor to switch between files and previewed the source code. 

CleanShot 2021-01-18 at 17 04 30

But by setting up the other way:

```vim
if executable('fish')
    set shell=fish
    let $SHELL="zsh
endif
```

I got no loading time when moving cursor between file names and still can have fish as my internal shell.

**Comment by eddyekofo94** [comment]
@yuki-yano well I believe it is better to have it working straight out of the box, plus setting fish as my default shell breaks some other configs.

**Comment by yuki-yano** [comment]
Sorry for the delay in replying.

I understand, I will try to incorporate this change.
Can you add a note about the change in the doc as well?

Also, is it necessary to add line breaks?
As long as I'm using markdownlint, it doesn't seem to be a problem.

File: plugin/fzf_preview.vim [hunk 1]
```diff
@@ -33,7 +33,11 @@ if !exists('g:fzf_preview_command')
 endif
 
 if !exists('g:fzf_preview_if_binary_command')
-  let g:fzf_preview_if_binary_command = '[[ "$(file --mime {})" =~ binary ]]'
+  if executable('fish')
```
Comment (yuki-yano) [inline plugin/fzf_preview.vim:36]:
```suggestion
  if &shell ==# 'fish' && executable('fish')
```

**Review by yuki-yano — CHANGES_REQUESTED** [review]

**Pull request closed without merging**